lib/src/test/java/uk/gov/di/data/lep/library/services/MapperTest.java

```diff
@@ -74,4 +78,52 @@ void xmlMapperSerialisesXmlToObject() throws JsonProcessingException {
         assertEquals(expected.gender, result.gender);
         assertEquals(expected.verificationLevel, result.verificationLevel);
     }
+
+    @Test
+    void objectMapperMapsCompleteIsoDateCorrectly() throws JsonProcessingException {
```

Comment:
These tests might make more sense on the serializers themselves, now they're not part of the Mapper config, but this is a very weak feeling.